.travis.yml

```diff
@@ -20,6 +20,11 @@ addons:
 before_install:
   - sudo apt-get update -qq
 install:
+  - if [ "$TRAVIS_BUILD_DIR" != "$GOPATH/src/github.com/openebs/lvm-localpv" ]; then
```

Comment:
Why do we need the travis workflow. @pawanpraka1 ?

Reply:
To run the test cases on the forked repo.

Reply:
Can we remove them and use github actions on the fork also. IMHO, keeping it just for running tests on fork doesn't sound like a great way!

Reply:
can we setup minikube via github action?

Reply:
Last time I tried, minikube was not working. If we need minikube for testing then we should keep travis. I thought it was only for unit tests.